gstats/gstats.c

```diff
@@ -958,7 +958,7 @@ static int SendChallengeResponse(const char *indata, int gameport)
 {
 	static char challengestr[] = {'\0','h','a','l','l','e','n','g','e','\0'};
 	char *challenge;
-	char resp[128];
+	char resp[256];
```

Comment:
384 (better if this would be modified in the future to compute the correct key length)

Reply:
https://github.com/GameProgressive/UniSpySDK/pull/27/commits/dd1ea8b03016b32abcb8f71cf8f4ef6792440cd8